src/run/parser/source_live_timer.rs

```diff
@@ -0,0 +1,106 @@
+//! Provides the parser for the SourceLiveTimer splits files.
+
+#![allow(non_snake_case)]
+
+use std::io::Read;
+use std::result::Result as StdResult;
+use serde_json::de::from_reader;
+use serde_json::Error as JsonError;
+use {GameTime, Run, Segment, Time, TimeSpan};
+
+quick_error! {
+    /// The Error type for splits files that couldn't be parsed by the
+    /// SourceLiveTimer Parser.
+    #[derive(Debug)]
+    pub enum Error {
+        /// Failed to parse JSON.
+        Json(err: JsonError) {
+            from()
+        }
+    }
+}
+
+/// The Result type for the SourceLiveTimer parser.
+pub type Result<T> = StdResult<T, Error>;
+
+#[derive(Deserialize)]
+struct Splits {
+    Category: String,
+    RunName: Option<String>,
+    Splits: Option<Vec<Split>>,
+}
+
+#[derive(Deserialize)]
+struct Split {
+    Map: String,
+    Name: Option<String>,
+    #[allow(dead_code)]
+    Ticks: Option<u64>,
```

Comment:
If it's dead code, then I'd prefer it to be removed or even better, just commented out.

Reply:
Only had this field here because of serde needing it for deserialization. Forgot there was an ignore tag

Reply:
doesn't it just ignore fields that you don't mention?

Reply:
Maybe? I remember when I tried (like 3 versions ago) it would throw an error, Expected ...